src/Controller/ProductImagesController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace App\Controller;
+use App\Entity\Image;
+use App\Entity\Product;
+use App\Form\ImageType;
+use App\Service\FileUploader;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+/**
+ * @Route("/product")
+ * @property FileUploader fileUploader
+ */
+class ProductImagesController extends AbstractController
+{
+    public function __construct(FileUploader $fileUploader)
+    {
+        $this->fileUploader = $fileUploader;
```

Comment:
jeśli masz klasę, która rozszerza inną klasę, to w konstuktorze dziecka powinno zostać wywołanie construkta rodzica- parent::_construct(...), w przeciwnym razie rodziic może przestać działać, bo np. nie będzie miał ustawionych parametrów klasy